docs/reference/query-dsl/range-query.asciidoc

```diff
@@ -21,147 +23,206 @@ GET _search
         }
     }
 }
---------------------------------------------------
+----
 // CONSOLE 
 
-The `range` query accepts the following parameters:
+[[range-query-top-level-params]]
+==== Top-level parameters for `range`
+
+`<field>`::
++
+--
+Field you wish to search.
+--
+
+[[range-query-field-params]]
+==== Parameters for `<field>`
+
+`gt`::
+Greater than. Optional.
+
+`gte`::
+Greater than or equal to. Optional.
+
+`lt`::
+Less than. Optional.
+
+`lte`::
+Less than or equal to. Optional.
+
+`format`::
++
+--
+Date format used to convert `date` values in the query.
+
+By default, {es} uses the <<mapping-date-format,date `format`>> provided in the
+`<field>`'s mapping. This value overrides that mapping format.
+
+For valid syntax, see <<mapping-date-format,`format`>>. Optional.
+
+[WARNING]
+====
+If a `format` and `date` value are incomplete, {es} replaces any missing year,
+month, or date component with the start of
+https://en.wikipedia.org/wiki/Unix_time[Unix time], which is January 1st, 1970.
+
+For example, if the `format` value is `dd`, {es} converts a `gte` value of `10`
+to `1970-01-10T00:00:00.000Z`.
+====
+
+--
+
+[[querying-range-fields]]
+`relation`::
++
+--
+Indicates how the range query matches values for `range` fields. Optional. Valid
+values are:
+
+`INTERSECTS` (Default)::
+Matches documents with a range field value that intersects the query's range.
+
+`CONTAINS`::
+Matches documents with a range field value that entirely contains the query's range.
+
+`WITHIN`::
+Matches documents with a range field value entirely within the query's range.
+--
+
+`time_zone`::
++
+--
+https://en.wikipedia.org/wiki/List_of_UTC_time_offsets[Coordinated Universal Time
+(UTC) offset] used to convert `date` values in the query to UTC. Optional.
+
+Valid values are ISO 8601 UTC offsets, such as `+01:00` or -`08:00`.
```

Comment:
I think ISO 8601 does not include all the time zone ids that are possible here, especially not the ones with time zone changes on e.g. summer time (like "Europe/Berlin", "America/Toronto" etc...). Since its not obvious that those are allowed here, I think we should mention at least one of them. I don't know if there is a comprehensive list of all the IANA time zone ids that Java allows, but maybe https://en.wikipedia.org/wiki/List_of_tz_database_time_zones is a good enough link? Will check if those are mostly accurate in terms of what Javas time library allows.

Reply:
Just did some random checks between what `ZoneId.getAvailableZoneIds()` (should be what the java time library accepts) reports and whats on the liked wikipedia list, seems quite accurate to me atm.

Reply:
Thanks for the review @cbuescher. I documented support for IANA time zone IDs with d508e2d.